core/src/main/java/org/apache/iceberg/MetricsUtil.java

```diff
@@ -174,6 +175,8 @@ public static MetricsModes.MetricsMode metricsMode(
                           field.type(), file.upperBounds().get(field.fieldId()))));
 
   public static final String READABLE_METRICS = "readable_metrics";
+  public static final String REF_SNAPSHOT_ID = "reference_snapshot_id";
```

Comment:
I feel like this constant is not too suitable in this file, which is for Metrics.  I think if we have to have a constant, should be in BaseMetadataTable.  Else we can just actually hard code it, it seems simpler.

Reply:
I still feel we should move this one.